.github/workflows/release.yml

```diff
@@ -65,7 +65,19 @@ jobs:
       with:
         gradle-executable: "./test_runner/gradlew"
         arguments: "-p test_runner clean build shadowJar"
-        
+
+    - name: Gradle Integration Tests Android
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-executable: "./integration_tests/gradlew"
+        arguments: "-p integration_tests test --tests IntegrationTests.shouldMatchAndroidSuccessExitCodeAndPattern -Dflank-path=../test_runner/build/libs/flank.jar -Dyml-path=./src/test/resources/flank_android.yml"
+    
+    - name: Gradle Integration Tests iOS
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-executable: "./integration_tests/gradlew"
+        arguments: "-p integration_tests test --tests IntegrationTests.shouldMatchIosSuccessExitCodeAndPattern -Dflank-path=../test_runner/build/libs/flank.jar -Dyml-path=./src/test/resources/flank_ios.yml"
+    
```

Comment:
I think that they should be called on each PR (in `macOS` and `Ubuntu` workflows) to prevent merging code which brokes Flank behaviour.

WDYT?

Reply:
Good idea!

Reply:
Thanks, I moved integration tests from release to ubuntu/macos workflows